Changes.md

```diff
@@ -54,6 +54,7 @@ Breaking Changes
 - SceneTestCase : Removed `assertBoxesEqual()` and `assertBoxesAlmostEqual()` methods.
 - RenderController : Removed `pathForID()`, `pathsForIDs()`, `idForPath()` and `idsForPaths()`. Use `renderManifest()` instead.
 - Catalogue, CatalogueSelect, Display : Moved from GafferImage module to GafferScene.
+- CyclesOptions : Option plugs have been renamed to match the name of their option (e.g. `options.samples` is now `options.cycles:session:samples`). A compatibility config has been provided to allow CyclesOptions nodes to be loaded from scripts saved in earlier Gaffer versions.
```

Comment:
> Presets metadata registrations that rely on `GafferCycles` have been left in CyclesOptionsUI.py, but there might be a better place for them elsewhere?

One possibility would be to move them to `cyclesOptions.py`but register as function-based metadata that only imports IECoreCycles the first time the function is called. I'd be happy to defer this to a later PR though.

Reply:
Cool, I've added a todo for that in 3aeb468.